### ISSUE TITLE
gen-baitjs: Fix false positive workflow run

### ISSUE THREAD

**Issue by serkonda7** [tiabeast/bait#31]
### Description and Reproduction Sample

The follwoing run should have failed: https://github.com/tiabeast/bait/actions/runs/4941810231/jobs/8834792208

### Current Behavior

```
Cloning into '/home/runner/.bait/baitjs'...
/bin/sh: 1: Syntax error: Unterminated quoted string
error: cannot pull with rebase: Your index contains uncommitted changes.
error: please commit or stash them.
Everything up-to-date
(node:1805) [DEP014[7](https://github.com/tiabeast/bait/actions/runs/4941810231/jobs/8834792208#step:5:8)] DeprecationWarning: In future versions of Node.js, fs.rmdir(path, { recursive: true }) will be removed. Use fs.rm(path, { recursive: true }) instead
(Use `node --trace-deprecation ...` to show where the warning was created)
```

And exit with code 0.

### Expected Behavior

Exit with code 1 (--> fail).

### Environment details (bait doctor)

_no response_

**Comment by serkonda7** [comment]
I don't know why but it works like every second run :thinking:

**Comment by serkonda7** [comment]
Seems to be fixed with 45818a7691687c3719fbd9ce688d2dfd6bf51564
